Strategy11/formidable-forms
# Fix action limit message for actions with more than one limit

Fix https://github.com/Strategy11/formidable-pro/issues/4556

This could also be fixed in PHP by adding a new string for "99" action limit but still requires js update to check which action is clicked on and then showing the appropriate corresponding message but I believe we can limit the fix to js by manipulating the message as in this PR, which sounds less complex.

### Steps to test
1. Create a new form and add 99 'Confirmation' actions to the form. I used the following snippet to automate it:
```
$action_control = FrmFormActionsController::get_form_actions( 'on_submit' );
$i = 0;
while ( $i++ < 10 ) {
    $action_control->create( $form_id );
}
```
2. The icon for Confirmation action would be disabled when the limit is hit and clicking on it should show a message like in this screenshot:
<img width="798" alt="image" src="https://github.com/Strategy11/formidable-forms/assets/41271840/388e63a1-1cf7-4e0b-b3ea-dc08338bf623">
